### PR TITLE
Made a new goes page with updated information for new GOES E/W products.

### PR DESCRIPTION
**Pull request by tiffanycmeyer13**
-Added a lot more RGB products all with images
-Updated the GLM area
-Added new section for DMW
-Added new section for vertical temp/moisture profiles
-Specified which sectors had derived products
-Updated all images

@srcarter3 I'm still working on updating the pqact section, I'll commit that later today, but let me know on the rest. Do you think I should add images to the GLM section?

### CODE REVIEW

File: docs/cave/goes-satellite.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
+# GOES 16/17
+
+The goesr EDEX decoder supports the ingest of GOES products coming over NOAAPort and Unidata's IDD. These include single channel imagery, derived products (Level 2b netCDF files), gridded Geostationary Lightning Mapper (GLM) products (produced by Eric Bruning at Texas Tech), CIRA created RGB specific products, and vertical temperature/moisture profiles. Using derived parameters, additional RGB and channel difference products can be loaded. The dmw EDEX decoder supports the ingest of GOES derived motion winds.
```
Comment (srcarter3) [inline docs/cave/goes-satellite.md]: I know the repo is called 'goesr', but I think for this paragraph, for human readability it should say "The GOES-R EDEX decoder...".
Also, the single channel imagery, derived products, glm, cira and profiles should have internal links to their corresponding sections below (for CIRA just link to the first product).

File: docs/cave/goes-satellite.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+* Night Fog (10.3 - 2.9 &mu;m)
+* Day Fog (3.9 - 10.3 &mu;m)
+* Split Fire (2.2 - 1.6 &mu;m)
+* Split Ozone (9.6 - 10.3 &mu;m)
+* Split Water Vapor (6.19 - 7.3 &mu;m)
+* Split Snow (1.6 - 0.64 &mu;m)
+* Vegetation (0.64 - 0.87 &mu;m)
+* Upper Level Info (11.2 - 6.19 &mu;m)
+
+!!! note "The rendering of these products uses the **Jep** package in Python, which has known to have issues installing on the MacOS and Windows versions."
```
Comment (srcarter3) [inline docs/cave/goes-satellite.md]: I'm not sure I'd word it quite this way... Maybe say which need to be installed using very specific instructions found on our [installation page](link to our installation page).

File: docs/cave/goes-satellite.md [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+
+---
+
+## Derived Products
+
+Derived products are also known as **Level 2+** products.  Currently there's only derived products from GOES East available in AWIPS. Each sector has a different set of products available. To find out some more information on some of the products please the [**Quick Guides**](http://rammb.cira.colostate.edu/training/visit/quick_guides/) compiled by CIRA.
```
Comment (srcarter3) [inline docs/cave/goes-satellite.md]: "Currently there are only..."

File: docs/cave/goes-satellite.md [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+
+!!! note "These may not all be available for each sector."
+
+The current products offered in CAVE are and to the right is which GOES East sector they are available for (F=Full Disk, C=CONUS, M=Mesoscale):
```
Comment (srcarter3) [inline docs/cave/goes-satellite.md]: I think this sentence is incomplete?  "The current products offered in CAVE are ? and to the right...)

**Review by srcarter3 — COMMENTED** [review]
Just a couple small things.

File: docs/cave/goes-satellite.md (hunk 1 above)
Comment (tiffanycmeyer13) [reply docs/cave/goes-satellite.md]: I could color code/bold/quote the "goesr" - I was trying to use the actual name of the edex decoder that is being used - and similarly in the last paragraph the "dmw" EDEX decoder - that's the name of the EDEX decoder.

The paragraph below has all of the internal links (this was just how it was before....) but I can change it to be just in this first paragraph

File: docs/cave/goes-satellite.md (hunk 2 above)
Comment (tiffanycmeyer13) [reply docs/cave/goes-satellite.md]: Sure, I only updated the "formatting" of the note, the wording was what existed previously. Do we have problems with jep in Mac now - I didn't think so? New wording:

"The rendering of these products uses the **Jep**  package in Python, which has specific installation steps (link to instructions) for Windows"

File: docs/cave/goes-satellite.md (hunk 4 above)
Comment (tiffanycmeyer13) [reply docs/cave/goes-satellite.md]: okay, I'll fix it haha. thanks

**Pull request closed without merging**